modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/display/context/builder/IndexActionsDisplayContextBuilder.java

```diff
@@ -126,6 +130,10 @@ private JSONArray _getVirtualInstancesJSONArray() {
 		return jsonArray;
 	}
 
+	private boolean _isConcurrentModeSupported() {
+		return _searchCapabilities.isConcurrentModeSupported();
+	}
```

Comment:
Since this method only has 1 line and is already readable as "isConcurrentModeSupported", let's remove this `_isConcurrentModeSupported()` method and inline it directly in `getData()`.